.github/scripts/install-arduino-ide.sh

```diff
@@ -210,7 +212,13 @@ function build_sketches() # build_sketches <fqbn> <examples-path> <chunk> <total
         || [ "$sketchnum" -gt "$end_index" ]; then
         	continue
         fi
-        build_sketch "$fqbn" "$sketch" "$xtra_opts"
+		local sketchBuildFlags=""
+		if [ -f "$sketchdir/.test.build_flags" ]; then
+			while read line; do
+				sketchBuildFlags="$sketchBuildFlags $line"
+			done < "$sketchdir/.test.build_flags"
+		fi
```

Comment:
please care for tabs/spaces. in other words... do not use tabs :)

Reply:
Replaced all the tabs with spaces